apps/api-frontend/src/app/pages/search-result.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState, useEffect } from 'react';
+import { environment } from '../../environments/environment';
+import { ParsedArticleParagraphStandalone } from '@foodmedicine/interfaces';
+import ClimbingBoxLoader from 'react-spinners/ClimbingBoxLoader';
+import { useLocation, useHistory } from 'react-router-dom';
+import { onSearch } from './onsearch';
+import './search-result.css';
+import { SearchBar } from '@foodmedicine/components';
+
+function SingleResult(props: {
+  paragraph: ParsedArticleParagraphStandalone;
+  key: string;
+}) {
+  return (
+    <div
+      className="single-result-container"
+      aria-label="search-result"
+      onClick={() => window.open(props.paragraph.head.xmlFullTextDownloadLink)}
+    >
+      <h4 aria-label="paper's title">{props.paragraph.head.title}</h4>
```

Comment:
You shouldn't need an aria label here at all. An aria-label is generally used to give something a text alternative that would not otherwise be easily understood by something like a screen reader (an app which will "speak out loud" the elements of a page for non-sighted users). So aria labels are good for things like an image or icon, but not necessary for paragraphs / text as those are (generally) parsed automatically.